components/toggle/theme.css

```diff
@@ -0,0 +1,363 @@
+@import 'variables.css';
```

Comment:
You need to import **variables** here, since you're not using any from this file.